### PR TITLE
Upgrade scss lint

### PR DESCRIPTION
**Pull request by arunagw**
* Locked to specific version to avoid upgrade problems
* Update configs from scss-lint default file.

### CODE REVIEW

**Comment by gylaz** [comment]
Looks good. Does this fix the `bubbles?` problem?

**Comment by salbertson** [comment]
@gylaz the build is failing, so I'm guessing no.

**Comment by teoljungberg** [comment]
`bubbles?` is sadly still there, but it's nice to keep the gem up to date anyhow

File: Gemfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ gem "resque-sentry"
 gem "resque-timeout"
 gem "rubocop", "0.29.1"
 gem "sass-rails"
-gem "scss-lint", require: false
+gem "scss-lint", "0.34.0", require: false
```
Comment (gylaz) [inline Gemfile]: What is the reason to lock this explicitly? Shouldn't we just monitor `Gemfile.lock` changes in pull requests to make sure the version doesn't get bumped there?

Comment (teoljungberg) [inline Gemfile]: There was a breakage between `0.31.0` and `0.34.0` that broke the gem unless you added the new configurations. So to clarify that this gem might be risky to upgrade we added the lock in

**Comment by croaky** [comment]
Looks good.

**Comment by gylaz** [comment]
This should help us with the background jobs timing out. It looked like we were hitting an infinite loop situation which was fixed in `scss-lint` recently: https://github.com/causes/scss-lint/commit/d8be7c5125e68b963111a538c162b425cddf3e2e#diff-11767ea2aca3dca34f9d7d2cc03c73a8

**Pull request merged**